week-07/day-5/FoxClub/FoxClub/Views/Home/Index.cshtml

```diff
@@ -0,0 +1,32 @@
+﻿@model List<FoxClub.Models.Fox>
+
+<a href="/Login">Login</a>
+<br />
+<br />
+
+<img src="~/images/greenfox.png">
+
+@foreach (var value in Model)
```

Comment:
`var fox in Model`

Reply:
actually, most likely you'll always have a list of just one fox, if you had that `Find` in the service, you'd pass just one fox into the View and you woulnd't have to loop through the list